Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,362 @@
+| Authentication of users             | ✓ Local database, LDAP, OIDC, UAA             | ✓ Local database, LDAP, Keystone, JWT                               | ✓ Local database             |
+| Authorization                       | ✓                                             | ✓                                                                   | ✓                            |
+| Automation                          | ✓ Webhooks                                    | ✓ Webhooks, building images                                         | ✗                            |
+| Vulnerability scanning              | ✓ Trivy, Clair                                | ✓ Clair                                                             | ✗                            |
+| Content Trust and Validation        | ✓ Notary, Cosign                              | ✓ Cosign                                                            | ✗                            |
+| Multi-tenancy                       | ✓ (not on the storage level)                  | ✓ (not on the storage level)                                        | ✓ (not on the storage level) |
+| Backup and restore                  | ✓                                             | ✓                                                                   | ✗                            |
+| Monitoring                          | ✓ Prometheus metrics                          | ✓ Prometheus metrics                                                | ✓ Prometheus metrics         |
+| HA mode                             | ✓                                             | ✓                                                                   | ✗                            |
+| Registry replication                | ✓                                             | ✓                                                                   | ✓                            |
+| Proxy cache                         | ✓                                             | ✓ Feature is in the technology preview stage (non production ready) | ✗                            |
+| Quota management                    | ✓ Based on storage consumption or image count | ✓ Based on storage consumption                                      | ✗                            |
+| Garbage collection                  | ✓                                             | ✓                                                                   | ✗                            |
+| Retention policy                    | ✓ Multiple tag retention rules                | ✓ Only tag expiration rules                                         | ✗                            |
+| Additional supported artifacts      | ✓ non-OCI Helm charts (ChartMuseum)           | ✗ (only OCI artifacts)                                              | ✓ Maven, YUM                 |
```

Comment:
Harbor is in deprecation process for Chartmuseum integration

https://github.com/goharbor/harbor/releases/tag/v2.7.0

Reply:
Hello @larsl-net 

Thank you for pointing that out!  
Harbor started the deprecation process also for Notary (from v2.6.0). 

We will adjust the document accordingly.

Reply:
comment has been addressed, hence this thread is marked as resolved